docassemble/AssemblyLine/al_general.py

```diff
@@ -389,6 +440,22 @@ def normalized_address(self) -> Union[Address, "ALAddress"]:
             return self.norm_long
         return self
 
+    def state_name(self, country_code=None):
+        if country_code:
+            return state_name(self.state, country_code=country_code)
+        # Do a quick check for a valid ISO country code (alpha-2 or alpha-3)
+        if hasattr(self, "country") and self.country and len(self.country) < 4:
```

Comment:
Does `alpha-3` actually work? Documentation just says alpha-2, but haven't checked source for that yet.

Reply:
Doesn't actually look like it will with the way that `state_name()` function is written, although pycountry does work with alpha_3 in general. This check just skips one try...except though.